jwst/resample/resample_utils.py

```diff
@@ -303,3 +303,28 @@ def decode_context(context, x, y):
         )
 
     return idx
+
+
+def _xy_range_from_bbox(xmin, xmax, ymin, ymax, shape, bbox=None):
+
+    if (xmin is xmax is ymin is ymax) and xmin in (0, None):
```

Comment:
The default value of `xmin`, `xmax`, `ymin`, `ymax` is 0. Why is this check for `None`?
Shouldn't the default be `None`? 
Or why not use an existing function to be consistent?
 https://github.com/spacetelescope/jwst/blob/675cf4a9c677c472072cfa2ea45bc28f50636a2a/jwst/assign_wcs/util.py#L902

Reply:
> Or why not use an existing function to be consistent?

Because this is the "inverse" function of `wcs_bbox_from_shape()`.

> Why is this check for None. Shouldn't the default be None?

I don't know. I didn't write this code. I do not think there is need to allow `None` but I also do not see harm in it. `cdrizz` uses the entire input image when `xmin=xmax=ymin=ymax=0` (both in `drizzle` and in `drizzlepac`). It does not accept `None` even though, IMO, it would be a more natural way to say "no limits - use the entire image". However, the way this is done here is inconsistent (again) with `dodrizzle()` in `gwcs_drizzle` module - that one uses 0 exclusively.